### DISCUSSION TITLE
acesso a inserção de novos Parlamentares sem acesso as tabelas auxiliares, e facilitar a inserção

### DISCUSSION THREAD

**Discussion by Danpeli** [interlegis/sapl#3311] [Q&A] [answered]
## Comportamento Esperado

Atribuir a um usuário a inserção de novos Parlamentares, sem que este usuário tenha acesso às tabelas auxiliares

## Comportamento Atual

Não há Grupo disponível apenas para a inserção de parlamentares, sendo atribuido o grupo **Operador Geral**, o que dá ao usuario acesso às tabelas auxiliares, e ainda caso este tenha acesso a inserção de novo Perlamentar, esta operação só é possivel através das tabelas auxiliares ou de dentro de um cadastro de parlamentar, não é disponibilizado um botão para cadastro na listagem de parlamentares. 

## Possível Solução

1. Criar novo Grupo  com todas as permissões do modulo parlamentar;
2. Inserção do botão "Cadastrar Parlamentar" na listagem de Parlamentares:
    2.1.  sapl/sapl/templates/parlamentares/parlamentares_list.html : o trecho compreendido entre as linha 12 e 19 ganha um item abaixo da linha 16:
~~~
    <div class="actions btn-group float-right btn-group-sm" role="group">
        <a v-if="is_pesquisa" v-on:click="pesquisaChange" class="btn btn-primary text-white">Listar Parlamentar</a>
        <a v-else v-on:click="pesquisaChange" class="btn btn-outline-primary text-info">Pesquisar Parlamentar</a>
        {% if not request.user.is_anonymous  %}
            <a href="{% url 'sapl.parlamentares:vincular_parlamentar' %}" class="btn btn-outline-primary">Vincular Parlamentar</a>
            <a href="{% url 'sapl.parlamentares:parlamentar_create' %}"class="btn btn-outline-primary">Cadastrar Parlamentar</a>
        {% endif %}
        {% block more_buttons %}{% endblock more_buttons %}
    </div>
~~~

## Contexto

A dica acima visa disponibilizar o botão  para usuario logado, na página de listagem dos parlamentares sem necessidade de se entrar em qualquer cadastro.
## Imagens do Ocorrido

[Telas do SAPL](https://imgur.com/a/XD0jcik)

**Comment by LeandroJatai** [comment, marked as answer]
Veja Vídeo: https://youtu.be/9pq_JFDnbFg

**Comment by Danpeli** [comment]
> Veja Vídeo: https://youtu.be/9pq_JFDnbFg

A inserção de novo grupo de usuários e a ocorrência de um migrate com atualização futura do SAPL, não haveria nenhum problema para a base, somente a necessidade de se reinserir o grupo criado e relaciona-lo aos usuários?

**Comment by LeandroJatai** [comment]
não não... seus grupos são mantidos e não são alterados... os migrates não mexem nos seus grupos!!! ... só não adianta vc mexer nos grupos padrão... mas você pode inclusive abandonar esses grupos e criar os seus próprios...
